### PR TITLE
linking #9

### PR DESCRIPTION
**Pull request by CubingPanda**
Signed-off-by: CubingPanda <aayushsingh441@gmail.com>

sdjgfdsklb

### CODE REVIEW

File: panda.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <p>slkfdjhsdjkfhdsvb</p>
+    <ul>
+        <li>waoh!!!!!!</li>
+        <li>xD</li>
+        <li>sup bois???</li>
+    </ul>
```
Comment (samarasimha302) [inline panda.html:14]: use ordered list here

**Review by samarasimha302 — CHANGES_REQUESTED** [review]
use ordered list here

File: panda.html (hunk 1 above)
Comment (CubingPanda) [reply panda.html]: i have changed it,.... you can now close the issue.

**Review by samarasimha302 — APPROVED** [review]
good boy

**Review by StealthAdder — APPROVED** [review]
Code works! good job!

**Comment by CubingPanda** [comment]
it solves the issue

**Comment by StealthAdder** [comment]
Cool then i'l merge the PR

**Pull request merged**